geocube/api/core.py

```diff
@@ -59,7 +59,7 @@ def make_geocube(
             gcds = make_geocube(vector_data='my_vector.geopackage', like=other_gcds)
 
     fill: float, optional
-        The value to fill in the grid with for nodata. Default is -9999.0.
+        The value to fill in the grid with for nodata. Default is NaN.
```

Comment:
It looks like the default is actually `None` in this function; do you want it to be `numpy.nan`?

Reply:
Hmmm, I think you have a good point here. It does get changed to `np.nan` later, but putting it here as well would be more clear.